packages/shared/src/components/post/MarkdownPostContent.tsx

```diff
@@ -19,11 +18,12 @@ function MarkdownPostContent({ post }: MarkdownPostContentProps): ReactElement {
       {post.type === PostType.Freeform && post.image && (
         <Link href={post.image}>
           <a target="_blank" rel="noopener noreferrer">
-            <Image
-              fetchPriority="high"
-              loading="eager"
-              src={post.image}
-              alt="Post cover image"
+            <LazyImage
```

Comment:
I actually vaguely remember we having some issues with this lazyImage and decided to slowly deprecate it in favor of the native lazy loading functionality.
Maybe @sshanzel knows best the in/outs.

Reply:
I see. The other post content types were already using LazyImage, and it has a placeholder/skeleton already built in, while Image does not. I chose to go this way since Image is used a lot of places and didn't want to create some unexpected visual bugs by introducing skeleton logic into it.

Could potentially create an Image component just for post cover images, though

Reply:
I think this one was refactored to use native lazy loading (from what I see in codebase)

Reply:
Any specific reason why only this content type ? @capJavert 

The differences between the results weren't significant, so if you think it's better to keep it on the Image component, that is fine.